chalk/shapes/__init__.py

```diff
@@ -43,34 +32,39 @@ def arc(radius: float, angle0: float, angle1: float) -> Diagram:
       Diagram
 
     """
-    return Path([ArcSegment(angle0, angle1 - angle0).scale(radius)]).stroke()
+    return (
+        ArcSegment(angle0, angle1 - angle0)
+        .at(V2.polar(angle0, 1))
+        .stroke()
+        .scale(radius)
+    )
 
 
-def polygon(sides: int, radius: float, rotation: float = 0) -> Diagram:
-    """
-    Draw a polygon.
+# def polygon(sides: int, radius: float, rotation: float = 0) -> Diagram:
+#     """
+#     Draw a polygon.
 
-    Args:
-       sides (int): Number of sides.
-       radius (float): Internal radius.
-       rotation: (int): Rotation in degress
+#     Args:
+#        sides (int): Number of sides.
+#        radius (float): Internal radius.
+#        rotation: (int): Rotation in degress
 
-    Returns:
-       Diagram
-    """
-    return Path.polygon(sides, radius, to_radians(rotation)).stroke()
+#     Returns:
+#        Diagram
+#     """
+#     return Trail.polygon(sides, radius, to_radians(rotation)).stroke()
 
 
 def regular_polygon(sides: int, side_length: float) -> Diagram:
-    return Path.regular_polygon(sides, side_length).stroke()
+    return Trail.regular_polygon(sides, side_length).stroke().center_xy()
```

Comment:
For rotations it might be more reasonable to center the polygon in its centroid (using the `centered` method on the trail) instead of the center of the bounding box (using the `center_xy` method on the diagram). That is,
```python
return Trail.regular_polygon(sides, side_length).centered().stroke()
```